### DISCUSSION TITLE
Is scrollofdebug compatible with RKPD2 ?

### DISCUSSION THREAD

**Discussion by hygfvtt463** [Zrp200/ScrollOfDebug#3] [General]
Hi Zrp ! I really love your RKPD2 and just wander is it possible to implement the scrollofdebug into it ?

**Comment by Zrp200** [comment]
yes. Actually, it's already implemented in RKPD2, and 2.0.0 is implemented indev in there as well. This applies to RKA as well, of course, since RKA is forked from RKPD2.
